integration-tests/oauth2-spring/src/main/java/com/okta/spring/tests/oauth2/spring/opaquetoken/BasicOpaqueTokenResourceServerApplication.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2021-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.spring.tests.oauth2.spring.opaquetoken;
+
+import com.okta.spring.boot.oauth.Okta;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@SpringBootApplication
+@EnableGlobalMethodSecurity(prePostEnabled = true, securedEnabled = true)
+public class BasicOpaqueTokenResourceServerApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(BasicOpaqueTokenResourceServerApplication.class, args);
+    }
+
+    @Configuration
+    static class OpaqueTokenResourceSecurityConfigurer extends WebSecurityConfigurerAdapter {
+
+        @Override
+        public void configure(HttpSecurity http) throws Exception {
+
+            Okta.configureResourceServer401ResponseBody(http);
```

Comment:
Does this test pass without this line?
(ideally it should be removed from this test)

Reply:
we can't remove it